composer.json

```diff
@@ -25,15 +25,15 @@
     ],
     "require": {
         "php": "^7.2|^8.0.2|^8.1",
-        "laravel/framework": "5.3.*|5.4.*|5.5.*|5.6.*|5.7.*|5.8.*|6.*|7.*|8.*|^9.0|^10.0",
-        "eklundkristoffer/seedster": "^7.0",
+        "laravel/framework": "5.3.*|5.4.*|5.5.*|5.6.*|5.7.*|5.8.*|6.*|7.*|8.*|^9.0|^10.0|^11.0",
+        "eklundkristoffer/seedster": "^8.0",
         "laravel/helpers": "^1.5"
```

Comment:
Might need to bump `laravel/helpers` up to [v1.7](https://packagist.org/packages/laravel/helpers#v1.7.0)

Reply:
^ for Laravel 11 support